app/plugin_api.go

```diff
@@ -124,10 +124,30 @@ func (api *PluginAPI) UpdateChannel(channel *model.Channel) (*model.Channel, *mo
 	return api.app.UpdateChannel(channel)
 }
 
+func (api *PluginAPI) AddChannelMember(channel *model.Channel, userId string) (*model.ChannelMember, *model.AppError) {
```

Comment:
Accepts a `*model.Channel` vs. just a `channelId` since that's what `AddChannelMember` takes, and I figured I wouldn't add a step to look up the channel here. Thoughts? I converged on `channelId` everywhere else.

Reply:
I guess it depends whether we think the API users are going to have the channel already most of the time. I'm slightly leaning towards using `channelId` and just adding the extra look-up here for consistency but only 1/5 on that so I'll leave it up to you